selfdrive/car/hyundai/values.py

```diff
@@ -62,6 +67,7 @@ class HyundaiFlags(IntFlag):
   CANFD_ALT_GEARS_2 = 64
   SEND_LFA = 128
   USE_FCA = 256
+  CAN_CANFD = 512
```

Comment:
Let's use the existing `CANFD_HDA2` for deciding HDA2-specific behavior, but just add a new flag for the half-CANFD cars